### DISCUSSION TITLE
Is it possible to switch fan curves when application (game) is launched?

### DISCUSSION THREAD

**Discussion by elpeterson** [Rem0o/FanControl.Releases#1606] [Q&A]
I have a mini-itx build with a very silent fan curve setup, under my normal work day I really prefer for a silent operation. However during gaming I'd like to instead have my minimum fan speed be 50% instead of 0%, or better yet, simply swap out a more aggressive fan profile while a set of applications is active. 

Is this something that can be accomplished?

**Comment by Rem0o** [comment]
Yes, with any hotkey/shortcut application, just run FanControl.exe -c configYouWant.json

**Reply by backslashV** [reply]
It'd be great if you could integrate this into the app itself.

**Reply by leftcoastfunk** [reply]
I agree that having the Fan Control app itself recognize a game launch in order to automate profile-change is highly preferable for those who regularly switch profiles and do so with many different applications. For a couple of applications the command line hook is useful but not the easiest thing for some to do

**Reply by leftcoastfunk** [reply]
> Yes, with any hotkey/shortcut application, just run FanControl.exe -c configYouWant.json

Another issue with this method is that there's no recognition of when the application in question *stops* running, forcing a manual change back to the previous profile

**Reply by Ryuji-Yatahagane** [reply]
Hi there,

I'm messing with the same. i'll like to change the profile by a Hotkey but i dont know how to execute that comand u guys give "FanControl.exe -c profile.json"
If i execute it on a CMD window after adding to the PATH variable the path of the .exe app it just do nothing. No error, but not changing the profile.

Do u knwo any reason ?

**Reply by Rem0o** [reply]
As a first test, just run a terminal window from the fancontrol folder, then do FanControl.exe -c the_config_file  where  the_config_file is a json file in your configuration folder.  No need to add the directory, just the config file, with proper casing.

**Reply by Ryuji-Yatahagane** [reply]
Hi,

Thanks for ur fast response.
I found the error, It was because the name of my profile.json has a blanc space on it MAX COOLER.json and  the command recognice as to diferents arguments, so i change it's name to MAXCOOLER.json and it works fine.

Thanks for the support

**Reply by Rem0o** [reply]
you can also do quotes when using a file name with spaces.

```powershell
.\FanControl.exe -c "file with spaces.json"
```
